<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Map Layer Fields</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Cell-sector Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QgsFieldExpressionWidget" name="wName"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string notr="true">Locked Cell-sector</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QgsFieldExpressionWidget" name="wLock">
       <property name="expressionDialogTitle">
        <string notr="true">Expression Builder</string>
       </property>
       <property name="allowEmptyFieldName">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Antenna Height</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QgsFieldComboBox" name="wHeight"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Azimuth</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QgsFieldComboBox" name="wAzimuth"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Azimuth Shift</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QgsFieldExpressionWidget" name="wShift"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>M - Downtilt</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QgsFieldComboBox" name="wM_Tilt"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>E - Downtilt</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QgsFieldComboBox" name="wE_Tilt"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Horizontal Beam Width</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QgsFieldExpressionWidget" name="wH_Width">
       <property name="allowEmptyFieldName">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Vertical Beam Width</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QgsFieldExpressionWidget" name="wV_Width">
       <property name="allowEmptyFieldName">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Min E-Downtilt</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QgsFieldExpressionWidget" name="wMin_E_Tilt">
       <property name="allowEmptyFieldName">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Max E-Downtilt</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QgsFieldExpressionWidget" name="wMax_E_Tilt">
       <property name="allowEmptyFieldName">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="lbError">
     <property name="toolTip">
      <string notr="true"/>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFieldComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsfieldcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsFieldExpressionWidget</class>
   <extends>QWidget</extends>
   <header>qgsfieldexpressionwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
